### DISCUSSION TITLE
Please help me guys for this project in Javascript using console

### DISCUSSION THREAD

**Discussion by musclexd** [community/community#122666] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can I display in the console if the cartQuantity value touches < 0, or it becomes -1, -2, -3 ... 
**I want to display in the console 'Cart Quantity: No more item in the cart!' instead of just counting -1, -2, -3 downwards.**

![111](https://github.com/community/community/assets/169036110/a7579aa9-959c-4f86-b50e-73263e711993)

here is the code:

<!DOCTYPE html>

<html>
  <head>
    <title>Cart Quantity</title>
  </head>

  <body>
    <p>Cart Quantity</p>

    <button onclick="
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">Show Quantity</button>

    <button onclick="
      cartQuantity ++;
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">Add to Cart</button>

    <button onclick="
      cartQuantity += 2;
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">+2</button>

    <button onclick="
      cartQuantity += 3;
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">+3</button>

    <button onclick="
      cartQuantity += 4;
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">+4</button>

    <button onclick="
      cartQuantity += 5;
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">+5</button>

    <button onclick="
      cartQuantity = 0;
      console.log('Cart was reset.');
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">Reset Cart</button>
    
    <button onclick="
      cartQuantity --;
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">Remove from cart</button>

    <button onclick="
      cartQuantity -= 2;
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">-2</button>

    <button onclick="
      cartQuantity -= 3;
      console.log(`Cart Quantity: ${cartQuantity}`);
    ">-3</button>
    
    <script>
      let cartQuantity = 0;
    </script>
  </body>
</html>

**Comment by CopperEagle** [comment, marked as answer]
Hey @musclexd 

The onclick attribute contains valid Javascript. So you can do whatever the language allows you to do. Here is a template for all your onclick attributes that *decrease* the `cartQuantity` value:

```js
if (cartQuantity <= 0){
    // for buttons that decrease cartQuantity
    console.log("Please don't create a vacuum in the cart...");
    // then don't modify the value
} else {
   console.log("Alright, modifying the number of items");
    // cartQuantity--; 
    // cartQuantity -= 2;  --> must change the if check to "if (cartQuantity <= 1)"
    // cartQuantity -= 3 --> "if (cartQuantity <= 2)"
}
```

Now you need to change the `else` branch to do the correct modification of the value for each onlick individually. Also, you need to change the `if` condition, see comments. The reset button can be just `cartQuantity = 0;`. For those buttons that increase the value, you can leave the code as is.

**Reply by musclexd** [reply]
Hey @CopperEagle , Thanks for the help and replying to my post. I will try this now.
